guides/hosting/infrastructure/filesystem.md

```diff
@@ -49,6 +49,26 @@ shopware:
       # The Adapter Configuration
 ```
 
+If you want to regulate the uploaded file types, then you could add the keys `allowed_extensions`for the public filesystem or `private_local_download_strategy` for the private filesystem.
+With the `private_local_download_strategy` key you could choose the download strategy for private files (e.g., the downloadable products):
+
+```yaml
+shopware:
+  filesystem:
+    public:
+      # The Adapter Configuration
+    private:
+      # The Adapter Configuration
+    allowed_extensions: # Array with allowed file extensions for public filesystem
+    private_allowed_extensions: # Array with allowed file extensions for private filesystem
+    private_local_download_strategy: # Name of the download strategy: php, x-sendfile or x-accel
+```
+
+Following download strategies are valid:
```

Comment:
can you mention when this download strategy will be used? e.g. that it is only relevant for download products

Reply:
See in line 53, is this enough? :thinking: